shutdown.sh

```diff
@@ -8,7 +8,8 @@ done
 
 if docker ps | grep hub-redis >/dev/null ; then
     docker stop hub-redis
-    docker rm hub-redis
+    # Docker removes hub-redis when it stops
```

Comment:
I think this should go in a separate PR. My quick internet searchings suggest that `stop` should not be doing an `rm` so maybe further investigation req'd.

Reply:
I am happy to move this into a different PR.  Its all to do with the way Docker is started by other scripts.  All the scripts start docker with `--rm` which tells docker to remove the container once its done.  Whoever made that change didn't update the shutdown scripts to reflect that change.  I'll move this to a new PR but we should probably settle on an approach to these scripts.